nitrite/src/main/java/org/dizitart/no2/mapper/JacksonMapper.java

```diff
@@ -1,233 +0,0 @@
-/*
```

Comment:
Let's not remove this as it will introduce breaking changes. Instead extends it from `GenericMapper` and keep `getObjectMapper()` exposed for now. May be add a deprecated annotation on top of that method and put a link to new facade for the user.